impl_core/src/main/java/io/opencensus/implcore/trace/export/SampledSpanStoreImpl.java

```diff
@@ -173,17 +173,13 @@ private Bucket getErrorBucket(CanonicalCode code) {
 
     private void considerForSampling(SpanImpl span) {
       Status status = span.getStatus();
-      // Null status means running Span, this should not happen in production, but the library
-      // should not crash because of this.
-      if (status != null) {
```

Comment:
Why does this method no longer need to check whether the span in running?

Reply:
My bad. Tried to do something else. Reverted